src/psyclone/psyad/transformations/preprocess.py

```diff
@@ -38,12 +38,14 @@
 translated to adjoint PSyIR.
 
 '''
-from psyclone.psyir.nodes import BinaryOperation, Assignment
+from psyclone.core import SymbolicMaths
+from psyclone.psyad.utils import node_is_active_names, node_is_passive_names
+from psyclone.psyir.nodes import BinaryOperation, Assignment, Range
 from psyclone.psyir.transformations import DotProduct2CodeTrans, \
     Matmul2CodeTrans, ArrayRange2LoopTrans, TransformationError
 
 
-def preprocess_trans(kernel_psyir):
+def preprocess_trans(kernel_psyir, active_variable_names):
     '''PSyclone kernel transformation script which replaces dotproduct and
```

Comment:
The docstring needs extending to describe the `expand` functionality.

Reply:
Extended.